src/bust.py

```diff
@@ -212,20 +217,40 @@ async def finish(self, say_goodbye: bool = True) -> None:
         self._finished = True
 
     async def play_song(self, index: int) -> None:
-        # Wait some time between songs
-        if config.seconds_between_songs:
-            embed_title = "Currently Chilling"
-            embed_content = "Waiting for {} second{}...\n\n**REMEMBER TO VOTE ON THE GOOGLE FORM!**".format(
-                config.seconds_between_songs,
-                "s" if config.seconds_between_songs != 1 else "",
-            )
-            embed = Embed(title=embed_title, description=embed_content)
-            await self.message_channel.send(embed=embed)
-            await asyncio.sleep(config.seconds_between_songs)
+        # Send the chilling message
+        embed_title = "Currently Chilling"
+        embed_content = (
+            "The track will start soon...\n\n**REMEMBER TO VOTE ON THE GOOGLE FORM!**"
+        )
+        embed = Embed(title=embed_title, description=embed_content)
+        await self.message_channel.send(embed=embed)
+        start_album_generation = time.time()
```

Comment:
Could you add a comment here explaining why we're using `time.time()`?

It's currently not clear just from reading the code.